website/docs/signing.md

```diff
@@ -15,6 +17,9 @@ targets: # Distro specific build requirements
         cmdline: "/signer"
 ```
 
+These are the targets that can leverage signing:
+`windowscross/container`, `windowscross/zip`, `mariner2/rpm`.
```

Comment:
can we say something like "at this time"?

re: windowscross/container, do we need to clarify that this is not container signing but artifact signing inside the container?